tests/integration/hooks/finding-post-processing.test.js

```diff
@@ -12,6 +12,7 @@ test(
 
     expect(count).toBe(2);
     expect(severities.high === 1)
```

Comment:
Does this test really fail if the condition evaluates to false? I've only ever seen expect be used together with a matcher.

I've only ever seen it used together with a matcher and the docs don't seem to say what happens when you call it by itself: https://jestjs.io/docs/en/expect#expectvalue

Reply:
You are right this was simply a mistake :+1: